src/main/java/org/infernalstudios/infernalexp/init/IEItems.java

```diff
@@ -136,4 +143,12 @@ public static void register(IEventBus eventBus) {
         return ITEMS.register(name, itemSupplier);
     }
 
+
+    public static <T extends Item> RegistryObject<T> registerItemConditioned(String name, Supplier<? extends T> itemSupplier, Supplier<? extends T> itemSupplierCompat, String modID) {
+            RegistryObject<T> item = ITEMS.register(name,ModList.get().isLoaded(modID)? itemSupplier:itemSupplierCompat);
+
+            return item;
+    }
+
+
```

Comment:
~~Skill~~ Formatting issue:
```suggestion
    public static <T extends Item> RegistryObject<T> registerItemConditioned(String name, Supplier<? extends T> itemSupplier, Supplier<? extends T> itemSupplierCompat, String modID) {
        RegistryObject<T> item = ITEMS.register(name, ModList.get().isLoaded(modID) ? itemSupplier : itemSupplierCompat);
        return item;
    }
```